packages/browser-destinations/destinations/evolv-ai/src/identifyUser/index.ts

```diff
@@ -0,0 +1,56 @@
+import type { BrowserActionDefinition } from '@segment/browser-destination-runtime/types'
+import type { Evolv } from '../types'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { formatAttributes } from '../utility'
+import { __createBinding } from 'tslib'
+import { setUser, setValues } from '../proxy'
+
+// Change from unknown to the partner SDK types
+const action: BrowserActionDefinition<Settings, Evolv, Payload> = {
+  title: 'Identify User',
+  description: `Send Segment user traits to Evolv AI`,
+  defaultSubscription: 'type = "identify"',
+  platform: 'web',
+  fields: {
+    attributes: {
+      description: 'JSON object containing additional attributes associated with the user.',
```

Comment:
```suggestion
      description: 'Object containing additional attributes associated with the user.',
```

Removing JSON just to avoid any confusion considering this will be mapped as an Object within Segment